File: 0231-power-of-two/0231-power-of-two.cpp
class Solution {
public:
    bool isPowerOfTwo(int n) {
        int i=0;

        while (true){
            if (pow(2,i)==n){
                return true;
            }
            else if (pow(2,i)>n){
                return false;

            }
            i++;
            }
        }   
};

File: 0231-power-of-two/0231-power-of-two.py
class Solution(object):
    def isPowerOfTwo(self, n):
        i=0
        while True:
            if 2**i==n:
                return True
            elif 2**i>n:
                return False
            i+=1
        """
        :type n: int
        :rtype: bool
        """
        